Doc/library/uuid.rst

```diff
@@ -12,8 +12,8 @@
 
 This module provides immutable :class:`UUID` objects (the :class:`UUID` class)
 and the functions :func:`uuid1`, :func:`uuid3`, :func:`uuid4`, :func:`uuid5`,
-:func:`uuid6`, and :func:`uuid8` for generating version 1, 3, 4, 5, 6,
-and 8 UUIDs as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
+:func:`uuid6`, :func:`uuid7`, and :func:`uuid8` for generating UUIDs version 1,
+3, 4, 5, 6, 7, and 8 as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
```

Comment:
```suggestion
:func:`uuid6`, :func:`uuid7`, and :func:`uuid8` for generating UUIDs for version 1
and 3-8 as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
```

or we don't need to repeat the numbers again:

```suggestion
:func:`uuid6`, :func:`uuid7`, and :func:`uuid8` for generating UUIDs for those versions,
as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
```

Reply:
Oh. I like the second formulation! WDYT @merwok?

Reply:
We are in the first lines of the doc. For readers it could be the first introduction to the concept of UUID versions, so I prefer the inelegant but clear enumeration (previous text) than a compact and less clear phrasing («for those versions» or «uuidN»).

But that doesn’t mean we need to write 1, 3, 4, 5, 6, 7, 8. Something better could like:
This modules provides a UUID class and functions for generating UUIDs corresponding to a specific UUID version as specified in the RFC: uuid1 for version 1, uuid3, uuid4, uuid5, [etc].

(I think enumerating the functions was done to provide a quick link to jump there, but we could replace that with a single link going to `The uuid module defines the following functions`.)

Reply:
> For readers it could be the first introduction to the concept of UUID versions

I haven't thought about this so thanks for the feedback. I think we can go with:

```rst
This modules provides a :class:`UUID` class and :ref:`functions <uuid-functions>` for
generating UUIDs corresponding to a specific UUID version as specified in the RFC, e.g.,
:func:`uuid1` for UUID version 1, :func:`uuid2` for UUID version 3, and so on (note that
UUID version 2 is deliberately omitted as it outside the scope of the RFC).
```